src/Symfony/Component/Scheduler/Trigger/CronExpressionTrigger.php

```diff
@@ -33,17 +54,54 @@ public function __toString(): string
         return $this->expression->getExpression();
     }
 
-    public static function fromSpec(string $expression = '* * * * *'): self
+    public static function fromSpec(string $expression = '* * * * *', ?string $context = null): self
```

Comment:
Do we need the context? Can't we just always use the message object string representation?

Reply:
I do this when using `RecurringMessage::cron()`. You're thinking to pass the message in here? I think it's good as-is so users's can use this manually if they can't/don't make their Message stringable.

Also, is `fromSpec` still an appropriate name?

Reply:
The `fromSpec` name seems ok to me.
Forcing messages to be stringable does not seem to be a huge constraint (as we don't care about the content itself).

Reply:
You want the type to be `?\Stringable $context = null`?